string name
float64[] joint_positions
float64[] joint_velocities